src/agent/onefuzz-supervisor/src/main.rs

```diff
@@ -84,12 +90,58 @@ fn version() {
 }
 
 fn licenses() -> Result<()> {
-    use std::io::{self, Write};
     io::stdout().write_all(include_bytes!("../../data/licenses.json"))?;
     Ok(())
 }
 
+fn redirect(opt: RunOpt) -> Result<()> {
+    let log_path = opt
+        .redirect_output
+        .expect("redirect should only be called with log_path");
+    info!("redirecting output: {}", log_path.display());
+
+    let stdout = OpenOptions::new()
+        .create(true)
+        .append(true)
+        .open(&log_path)
+        .context("unable to open log file")?;
+    let stderr = stdout.try_clone().context("unable to clone log handle")?;
```

Comment:
So, we're writing interleaving stderr/stdio. Maybe it would be better to write them to separate log files.

Reply:
(maybe better as a separate PR?)

Reply:
Updated.